### PR TITLE
feature/CD-42: added a /create route

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

File: src/common/components/Alert/Alert.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { ReactText, ReactChildren, ReactNode } from 'react';
```
Comment (CodeAnnihilator) [inline src/common/components/Alert/Alert.d.ts]: extra spaces in curly braces

File: src/common/components/Flex/Flex.d.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { ReactText, ReactNode, MouseEvent } from 'react';
```
Comment (CodeAnnihilator) [inline src/common/components/Flex/Flex.d.ts]: extra spaces in curly braces

File: src/common/components/Flex/Flex.d.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    direction?: 'row' | 'column' | string | null,
+    grow?: 1 | 2 | null,
+    height?: ReactText,
+    indent?: 'small' | 'medium' | 'large' | string | null,
+    justifyContent?: 'flex-end' | 'center' | 'space-between' | 'space-around' | string | null,
+    overflow?: 'hidden' | string | null,
+    shrink?: 0 | 1 | 2 | null,
+    width?: ReactText,
+    wrap?: 'wrap' | string | null,
+    /** Custom classes, that will be appended to component */
```
Comment (CodeAnnihilator) [inline src/common/components/Flex/Flex.d.ts]: this comment should be removed

File: src/common/components/Flex/Flex.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import cn from 'classnames';
+import React from 'react';
+
+import styles from './flex.scss';
+
+import IFlexProps from './Flex.d';
+/* tslint:disable */
```
Comment (CodeAnnihilator) [inline src/common/components/Flex/Flex.tsx]: why disable tslint here?

File: src/common/components/Flex/Flex.d.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    className?: string | null,
```
Comment (CodeAnnihilator) [inline src/common/components/Flex/Flex.d.ts:16]: can't be object here?

File: src/common/components/Flex/Flex.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+	height,
+	indent,
+	justifyContent,
+	overflow,
+	shrink,
+	onClick,
+	width,
+	wrap,
+	style,
+    className,
```
Comment (CodeAnnihilator) [inline src/common/components/Flex/Flex.tsx]: not aligned

File: src/common/components/Select/Select.d.ts [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import { ChangeEvent, ReactNode, MouseEvent } from 'react';
```
Comment (CodeAnnihilator) [inline src/common/components/Select/Select.d.ts]: extra spaces in curly braces

File: src/modules/CreateImage/components/CreateImageRoot.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+export default class CreateImageRoot extends PureComponent<IProps> {
+	public static defaultProps = {
+		step: 'initialization',
+	};
+
+	public render() {
+		return (
+			importsHash[this.props.step || 'initialization']
```
Comment (CodeAnnihilator) [inline src/modules/CreateImage/components/CreateImageRoot.tsx]: but we already have defaultProps step described. Why do we check for || ?

File: src/modules/CreateImage/constants/contants.ts [hunk 9]
```diff
@@ -0,0 +1,3 @@
+export const CHANGE_CURRENT_STEP = 'CHANGE_CURRENT_STEP';
```
Comment (CodeAnnihilator) [inline src/modules/CreateImage/constants/contants.ts]: should be 'CREATE_IMAGE/CHANGE_CURRENT_STEP' and etc.

File: src/modules/CreateImage/containers/CreateImageInitializationHeaderContainer.ts [hunk 10]
```diff
@@ -0,0 +1,10 @@
+import {connect} from 'react-redux';
+import {changeCurrentStep} from '../actions/actions';
+//tslint:disable
```
Comment (CodeAnnihilator) [inline src/modules/CreateImage/containers/CreateImageInitializationHeaderContainer.ts]: why do we need to disable tslint here?

**Review by CodeAnnihilator — COMMENTED** [review]
- selected element shouldn't be unselected when you click it in a list
- cross icon should be hidden when there is no text in a search bar
- header card text is not translating when changing language

**Review by CodeAnnihilator — COMMENTED** [review]
please resolve merge conflicts first

File: src/common/components/Flex/Flex.d.ts (hunk 5 above)
Comment (Gerasimov94) [reply src/common/components/Flex/Flex.d.ts:16]: no, it's just a string :)

File: src/modules/CreateImage/components/CreateImageRoot.tsx (hunk 8 above)
Comment (Gerasimov94) [reply src/modules/CreateImage/components/CreateImageRoot.tsx]: it's just for mock, don't care about that

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**